Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -272,11 +285,13 @@ def fetch_indicators(client: Client):
     '''
     args = {}
     last_run = demisto.getLastRun()
+    is_first_fetch = True
```

Comment:
first, fetch should be to determined by wether last_run exist or not

Reply:
Fixed it